### PR TITLE
[Task 17] Javadoc for convo list 

### PR DESCRIPTION
**Pull request by yiwxng**
What:

added class javadoc and some attribute javadocs in all of Conversation List files - explain in the CA terms.

Why:

the added javadoc convey clear information to TA and other group members who are working with conversation List about what CA layer is each class in. For group members, it also convey clear information about what each class is responsible for, in case they need to changes/ add changes in class.

Implementation detail:

for all classes and interfaces, javadoc is organized in a

- CA layer
- what is it doing?
- why does it exist?

### CODE REVIEW

**Review by navarhontes — APPROVED** [review]
Thanks for documenting!

**Review by candidcalcifer — DISMISSED** [review]

**Comment by jenniferjinyoungyang** [comment]
there is a merge conflict...

File: app/src/main/java/com/courseproject/tindar/ui/conversationlist/ConversationListFragment.java [hunk 1]
```diff
@@ -26,12 +26,18 @@
...
  */
 public class ConversationListFragment extends Fragment {
 
-    // TODO: Customize parameter argument names
     private static final String ARG_COLUMN_COUNT = "column-count";
+    private final int mColumnCount = 1;
+
 
     String userId;
     RecyclerView recyclerView;
```
Comment (jenniferjinyoungyang) [inline app/src/main/java/com/courseproject/tindar/ui/conversationlist/ConversationListFragment.java:43]: Can you add Javadoc for the attributes above?

File: app/src/main/java/com/courseproject/tindar/ui/conversationlist/ConversationListFragment.java [hunk 2]
```diff
@@ -44,7 +50,13 @@ public class ConversationListFragment extends Fragment {
     public ConversationListFragment() {
```
Comment (jenniferjinyoungyang) [inline app/src/main/java/com/courseproject/tindar/ui/conversationlist/ConversationListFragment.java:67]: Can you add Javadoc here for constructor?

File: app/src/main/java/com/courseproject/tindar/usecases/conversationlist/ConversationListInteractor.java [hunk 3]
```diff
@@ -22,11 +22,21 @@
 public class ConversationListInteractor implements ConversationListInputBoundary {
     private final ConversationListDsGateway conversationListDsGateway;
```
Comment (jenniferjinyoungyang) [inline app/src/main/java/com/courseproject/tindar/usecases/conversationlist/ConversationListInteractor.java:23]: Can you add JavaDoc for this attribute??

File: app/src/main/java/com/courseproject/tindar/ui/conversationlist/ConversationListFragment.java (hunk 1 above)
Comment (yiwxng) [reply app/src/main/java/com/courseproject/tindar/ui/conversationlist/ConversationListFragment.java:56]: done

**Review by jenniferjinyoungyang — APPROVED** [review]

**Review by navarhontes — APPROVED** [review]

**Review by candidcalcifer — APPROVED** [review]

**Pull request merged**